lib/typeform_data/requestor.rb

```diff
@@ -22,7 +22,8 @@ module Requestor
       Net::HTTPServiceUnavailable,
       Net::HTTPTooManyRequests,
       Net::HTTPBadGateway,
-      Net::HTTPGatewayTimeOut
+      Net::HTTPGatewayTimeOut,
+      Net::HTTPInternalServerError
```

Comment:
Pretty minor thing (I forgot this myself before)- but we should have a trailing comma here.